### DISCUSSION TITLE
Supabase with Prisma

### DISCUSSION THREAD

**Discussion by nehalist** [supabase/supabase#19214] [Questions]
I like the Prisma scheme generation; would it be bad practise to use Prisma to generate models, add RLS within Supabase and create the supabase migrations that way?

**Comment by jawngee** [comment]
Writing SQL would be faster and easier imho, ymmv.  You are essentially driving the long way around for no discernible gain.  You'd also be making it hard to use all of the dope features of postgres that prisma doesn't support or doesn't support well.
